extensions/ql-vscode/src/databases/config/db-config-store.ts

```diff
@@ -110,8 +113,15 @@ export class DbConfigStore extends DisposableObject {
     }
 
     const config: DbConfig = cloneDbConfig(this.config);
-    config.databases.remote.repositories.push(repoNwo);
-
+    if (parentList) {
+      config.databases.remote.repositoryLists.forEach((list) => {
```

Comment:
This is fine, but it will keep looping even after it found the list, which is a little strange. I don't expect it to be a problem because we don't anticipate large numbers of lists.

We could instead find the list with the given name, and update its `repositories`.

Reply:
👍